### ISSUE TITLE
Add an example app that is upgradeable using ZeppelinOS

### ISSUE THREAD

**Issue by snario** [counterfactual/contracts#60]
The apps in `appDefinitions` are natural candidates for ZeppelinOS upgradeable contracts. To exhibit this, it would be nice to show an example application in the contracts codebase using this.

**Comment by adklempner** [comment]
I'm interested in taking this on. Looks like `appDefinitions` is no more, would creating an upgradeable version of `HighRollerApp` or `TicTacToe` satisfy this requirement?

**Comment by snario** [comment]
Yes. This is kind of agnostic to the framework, moreso just to show what types of things could or should be used with ZeppelinOS.
